include/openthread.h

```diff
@@ -786,6 +787,69 @@ ThreadError otAddUnicastAddress(otInstance *aInstance, const otNetifAddress *aAd
 ThreadError otRemoveUnicastAddress(otInstance *aInstance, const otIp6Address *aAddress);
 
 /**
+ * This function pointer is called to create IPv6 IID during SLAAC procedure.
+ *
+ * @param[in]     aInstance  A pointer to an OpenThread instance.
+ * @param[inout]  aAddress   A pointer to structure containing IPv6 address for which IID is being created.
+ * @param[inout]  aContext   A pointer to creator-specific context.
+ *
+ * @retval kThreadError_None    Created valid IID for given IPv6 address.
+ * @retval kThreadError_Failed  Creation of valid IID for given IPv6 address failed.
```

Comment:
We're trying to move away from using generic failure codes like `kThreadError_Failed`.  Can we use a more specific error code here?  Feel free to define new error codes to `include/openthread-types.h`.